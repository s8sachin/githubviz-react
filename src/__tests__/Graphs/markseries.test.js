import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount, render, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Markserieschart from '../../components/Graphs/markseries';

configure({ adapter: new Adapter() });

describe('Markserieschart', () => {
it('should mount markseries component', () => {
    const wrapper = shallow(< Markserieschart />)
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})