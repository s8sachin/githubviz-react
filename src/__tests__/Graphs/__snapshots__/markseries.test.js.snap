// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markserieschart should mount sunburst component 1`] = `
<XYPlot
  className=""
  height={300}
  width={300}
>
  <VerticalGridLines
    attr="x"
    direction="vertical"
  />
  <HorizontalGridLines
    attr="y"
    direction="horizontal"
  />
  <XAxis
    attr="x"
    attrAxis="y"
    orientation="bottom"
  />
  <YAxis
    attr="y"
    attrAxis="x"
    orientation="left"
  />
  <MarkSeries
    className="mark-series-example"
    data={
      Array [
        Object {
          "size": 30,
          "x": 1,
          "y": 10,
        },
        Object {
          "size": 10,
          "x": 1.7,
          "y": 12,
        },
        Object {
          "size": 1,
          "x": 2,
          "y": 5,
        },
        Object {
          "size": 12,
          "x": 3,
          "y": 15,
        },
        Object {
          "size": 4,
          "x": 2.5,
          "y": 7,
        },
      ]
    }
    getNull={[Function]}
    opacity="0.8"
    sizeRange={
      Array [
        5,
        15,
      ]
    }
    strokeWidth={2}
  />
</XYPlot>
`;
